litgpt/scripts/download.py

```diff
@@ -45,8 +45,11 @@ def download_from_hub(
         print("\n".join(sorted(options, key=lambda x: x.lower())))
         return
 
-    if repo_id not in options:
-        print(f"Unsupported repo_id: {repo_id}. Please choose a valid repo_id from the following list:")
+    if model_name is None and repo_id not in options:
+        print(f"Unsupported repo_id: {repo_id}. If you are trying to download alternative "
+        "weights for a supported model, please specify the corresponding model via the `--model_name` option, "
+        "for example, `litgpt download NousResearch/Hermes-2-Pro-Llama-3-8B --model_name Llama-3-8B`."
+        "\nAlternatively, please choose a valid `repo_id` from the following list:")
         print("\n".join(sorted(options, key=lambda x: x.lower())))
```

Comment:
The list is going to be quite long, which will hide the text quite quickly.

Reply:
Maybe there will be a need to list it differently in the future.

Reply:
> The list is going to be quite long, which will hide the text very quickly.

That's true. How about the alternative message (I updated it in this PR):

> Unsupported repo_id: NousResearch/Hermes-2-Pro-Llama-3-8B. If you are trying to download alternative weights for a supported model, please specify the corresponding model via the `--model_name` option, for example, `litgpt download NousResearch/Hermes-2-Pro-Llama-3-8B --model_name Llama-3-8B`.
Alternatively, please choose a valid `repo_id` from the list of supported models, which can be obtained via
`litgpt download list`.

Reply:
Yep, that works